### PR TITLE
yt-dlp upgrade, non-web content detection tweak

### PR DESCRIPTION
**Pull request by matteocargnelutti**
- Upgraded to [yt-dlp 2023.06.21](https://github.com/yt-dlp/yt-dlp/releases/tag/2023.06.21)
- Tweaked non-web content detection to account for absence of content-type (assume target url is web content in that case)

### CODE REVIEW

File: Scoop.js [hunk 1]
```diff
@@ -690,8 +690,13 @@ export class Scoop {
       this.targetUrlContentType = contentType
     }
 
-    // If text/html, bail from non-web content capture process.
+    // If text/html or no content-type, bail from non-web content capture process.
     // Scoop.capture will go based on the value of `this.targeredUrlIsWebPage`.
```
Comment (bensteinberg) [inline Scoop.js:694]:
```suggestion
    // Scoop.capture will go based on the value of `this.targetUrlIsWebPage`.
```
(This is not part of the PR, but I just noticed this typo.)

**Review by bensteinberg — APPROVED** [review]
Looks good, I made a suggestion for an adjacent typo.

**Comment by matteocargnelutti** [comment]
Thanks @bensteinberg! Good catch, as always 🦅 👁️
